driver/bpf/maps.h

```diff
@@ -99,6 +99,15 @@ struct bpf_map_def __bpf_section("maps") stash_map = {
 };
 #endif
 
+#ifdef CAPTURE_SOCKETCALL
+struct bpf_map_def __bpf_section("maps") socketcall_args_map = {
```

Comment:
you need to add also an enum for this map https://github.com/falcosecurity/libs/blob/master/driver/bpf/types.h#L216-L230, otherwise, the userspace could parse them out of order. something like `SCAP_SOCKETCALL_MAP` should be ok

Reply:
Thanks @Andreagit97 . Should be solved with [this](https://github.com/falcosecurity/libs/pull/811/commits/528620fb6fd10a55f9b2ca538a4523ba472fa144) commit.